assembly/cli/src/site/markdown/index.md.vm

```diff
@@ -7,7 +7,7 @@ using the CLI frontend.
 
 #[[##]]# Configuring
 
-${docNameCap} CLI reads the same pom.xml configuration file that the Maven plugin version does.
+${docNameCap} CLI reads a configuration file (toolConfiguration.xml) that has the same structure as the Maven plugin.
```

Comment:
Does this include the whole pom.xml blabla of maven? Maybe make a full example what all of the pom file needs to be in the toolConfigurationFile.

Reply:
How it should look like is described in the Tool Configuration Readme (that's why I renamed it from pom.xml). But I could probably add a link to that readme to help the reader find it.